### PR TITLE
Fix olas links

### PR DESCRIPTION
**Pull request by jmoreira-valory**
- Fixed app domain names to olas.network.
- Housekeeping and fixed link-related issues.

> **Warning**
> **Ignore submodules in this PR.** Submodules have been updated only to avoid linter issues. The correct update of the submodules will take place in an upcoming PR.

### CODE REVIEW

File: price-oracle
Comment (Adamantios) [inline price-oracle:1]: Should be ebe74aafbed7fdc999c36f0ec581f74f57684080.

File: docs/protocol/mint_packages_nfts.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ In order to mint a software package as an NFT, you must ensure that you have:
 
 ## Mint a component
 
-Connect your wallet to the Protocol app, open the [components section](https://protocol.autonolas.network/components) and press the _Mint_ button. Fill in the data of your component:
+Connect your wallet to the {{ autonolas_protocol_registry_dapp }}, open the [components section](https://registry.olas.network/components) and press the _Mint_ button. Fill in the data of your component:
```
Comment (Adamantios) [inline docs/protocol/mint_packages_nfts.md:39]: Is there an option like:

```
[components section]({{ autonolas_protocol_registry_dapp }}/components)
```
?

If yes, then also `components` and `agents` sections can be replaced below.

Comment (jmoreira-valory) [reply docs/protocol/mint_packages_nfts.md]: Done

**Review by Adamantios — APPROVED** [review]

**Review by dagacha — APPROVED** [review]
I've checked a few links, just in case... 🙂

**Pull request merged**